enos/modules/vault_agent/templates/set-up-approle-and-agent.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+set -e
+
+binpath=${vault_install_dir}/vault
+
+fail() {
+  echo "$1" 1>&2
+  return 1
+}
+
+test -x "$binpath" || fail "unable to locate vault binary at $binpath"
+
+export VAULT_ADDR='http://127.0.0.1:8200'
+export VAULT_TOKEN='${vault_token}'
+
+$binpath auth disable approle || true
```

Comment:
Perhaps a comment explaining why we short circuit a possible error here.

Reply:
That's a great idea, let me add that